js/components/quack.jsx

```diff
@@ -16,10 +14,6 @@ var Quack = React.createClass({
         <div className="channel-box">
           <ChannelMessages>
           </ChannelMessages>
-          <ChannelUsers>
-          </ChannelUsers>
-          <Input>
-          </Input>
```

Comment:
it doesnt mess up the grid and I moved it in so that I could originally pass in the    `ChannelMessages#autoScroll` method into the `Input` react class as a prop, though that's unnecessary since we're going to autoscroll on any message as long as the user is already scrolled to the bottom.